src/sentry/static/sentry/app/components/dropdownReact.jsx

```diff
@@ -133,13 +148,6 @@ class DropdownReact extends React.Component {
     }
   };
 
-  // Control whether we should hide dropdown menu when it is clicked
-  handleDropdownMenuClick = e => {
-    if (this.props.keepMenuOpen) return;
```

Comment:
should this prop be removed from the codebase if it's no longer used?

Reply:
👍  since we do this by default now

Reply:
I think max means to remove it from propTypes + any parent components that render w/ it.